modules/ROOT/pages/apikit-for-soap-module-reference.adoc

```diff
@@ -24,7 +24,14 @@ Configuration element for a *SOAP Router*
 | SOAP version of the WSDL. |  | 
 | Address | String | Address of the web service. |  | 
 | MTOM Enabled | Boolean | Defines if the MTOM protocol manages the attachments. | `false` | 
-| Inbound Validation Enabled | Boolean | Defines if the schema of inbound request is validated. | `false` | 
+| Inbound Validation Enabled | Boolean | Defines if the schema of the inbound request is validated. | `false` | 
```

Comment:
You might consider something like

"Indicates whether the schema..."

Reply:
Thanks for your comments! 
I just try to be consistent with the structure used in the other descriptions.